site/package.json

```diff
@@ -25,6 +25,7 @@
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "unist-util-visit": "^2.0.3",
```

Comment:
@mtp1376 I can't comment on the file I want to, but why is there a `package-lock.json` file instead of updating the `yarn.lock` file in this PR?